references.bib

```diff
@@ -1,9 +1,91 @@
-@article{shannon, 
-	author={Shannon, Claude E.}, 
-	journal={The Bell System Technical Journal}, 
-	title={A mathematical theory of communication}, 
-	year={1948}, 
-	volume={27}, 
-	number={3}, 
-	pages={379-423}
+@misc{Manifesto,
+    title={Principles behind the Agile Manifesto},
+    author={{Kent Beck, Mike Beedle, Arie van Bennekum, Alistair Cockburn, Ward Cunningham, Martin Fowler, James Grenning, Jim Highsmith, Andrew Hunt, Ron Jeffries, Jon Kern, Brian Marick, Robert C. Martin, Steve Mellor, Ken Schwaber, Jeff Sutherland, Dave Thomas}},
+    howpublished={[Online]. Available: \url{http://agilemanifesto.org/principles.html}},
+    note={[Accessed: 22-Nov-2016]}
 }
+
```

Comment:
The Keith textbook may be worth citing to illustrate examples of practice within the games industry, rather than relying solely on the manifesto which will be more general than just games development.